### PR TITLE
Add Month and Weekview UG docs

### PR DESCRIPTION
**Pull request by Mahesh1772**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -19,26 +19,233 @@ To quickly get started with CLI-nton, follow these simple steps:
...
++---------------+---------------+---------------+---------------+---------------+---------------+---------------+
+|17             |18             |19             |20             |21             |22             |23             |
++---------------+---------------+---------------+---------------+---------------+---------------+---------------+
+|24             |25             |26             |27             |28             |29             |30             |
++---------------+---------------+---------------+---------------+---------------+---------------+---------------+
+|               |               |               |meeting        |               |               |               |
++---------------+---------------+---------------+---------------+---------------+---------------+---------------+
+|31             |               |               |               |               |               |               |
++---------------+---------------+---------------+---------------+---------------+---------------+---------------+
+```
```
Comment (clarencepohh) [inline docs/UserGuide.md:163]: Clear usage of code snippets to showcase outputs!

**Review by clarencepohh — COMMENTED** [review]
LGTM overall

File: docs/UserGuide.md [hunk 2]
```diff
@@ -19,26 +19,233 @@ To quickly get started with CLI-nton, follow these simple steps:
...
+The `next` command allows you to move to the next week or next month, depending on whether you are currently in the week or month view.
 
-Example of usage:
+**Usage:**
 
 ```
 next
 ```
 
+**Examples:**
```
Comment (kyhjonathan) [inline docs/UserGuide.md:34]: Comprehensive user guide, good job!

File: docs/UserGuide.md [hunk 3]
```diff
@@ -19,26 +19,233 @@ To quickly get started with CLI-nton, follow these simple steps:
...
+
+If you are currently in the **Week view:**
+```
+Week View: 31/03/2024 - 06/04/2024
```
Comment (kyhjonathan) [inline docs/UserGuide.md:38]: Perhaps could use pictures instead of typing out the output? Could feel "clunky" as a reader

**Review by kyhjonathan — COMMENTED** [review]
Overall LGTM. Perhaps could use pictures to represent outputs instead of typing out the CLI UI

**Pull request merged**